src/ProjectTask.php

```diff
@@ -317,6 +317,22 @@ public function post_addItem()
         /** @var array $CFG_GLPI */
         global $CFG_GLPI;
 
+        // Add team members
+        if (isset($this->input['teammember_list'])) {
+            $taskteam = new ProjectTaskTeam();
+            foreach ($this->input['teammember_list'] as $teammember) {
+                $teammember_info = preg_split("/[_-]/", $teammember);
+                $itemtype = substr(ucfirst($teammember_info[0]), 0, -1);
+                $taskteam->add(
+                    [
+                        'projecttasks_id' => $this->fields['id'],
+                        'itemtype'        => $itemtype,
+                        'items_id'        => $teammember_info[2]
+                    ]
+                );
+            }
```

Comment:
```suggestion
            $members_types = [User::getType(), Group::getType()];
            foreach ($members_types as $type) {
                $ids = ProjectTaskTeamDropdown::getPostedIds(
                    $this->input['teammember_list'],
                    $type
                );
                foreach ($ids as $id) {
                    $taskteam->add([
                        'projecttasks_id' => $this->fields['id'],
                        'itemtype'        => $type,
                        'items_id'        => $id
                    ]);
                }
            }
```

It would be best to reuse the existing `getPostedIds` method.
This avoid rewriting some custom code to parse the content and instead use one single centralized method.
This way if we need to change the used format, everything can be done in the reusable `AbstractRightsDropdown` class and others classes using its content do not need to be updated.

Note: there is an issue with my suggestion, which is the first line:
`$members_types = [User::getType(), Group::getType()];` 

It would be best to avoid manually defining this list.

Maybe the ProjectTask class could have a "public static getAssociableItemtypes()" method that return this available types (which could then be called here and in `ProjectTaskTeamDropdown::getTypes()`.

This avoid maintaining two separate lists.